### PR TITLE
Correct encrypt id token flow

### PR DESCRIPTION
**Pull request by AnjanaSamindraPerera**

### CODE REVIEW

File: en/docs/guides/login/oidc-token-encryption.md [hunk 1]
```diff
@@ -45,84 +45,36 @@ Make the following changes to the created service provider.
...
+    3. Open the created `key.pem` file using a text editor and you will see the extracted private key.
 
-    4.  Copy only the key string as shown in the sample below.
+    4. Copy only the key string as shown in the sample below.
 
         ![sample-key-string]({{base_path}}/assets/img/guides/sample-key-string.png)
+   
+    5. Paste the copied private key in the Client Private Key text area.
+   
+    6. Click Decrypt and the details of the decrypted ID Token will be displayed.
```
Comment (nilminiwso2) [inline en/docs/guides/login/oidc-token-encryption.md:76]:
```suggestion
    6. Click **Decrypt** and the details of the decrypted ID Token will be displayed.
```

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/guides/login/oidc-token-encryption.md [hunk 2]
```diff
@@ -45,84 +45,36 @@ Make the following changes to the created service provider.
...
-    4.  Copy only the key string as shown in the sample below.
+    4. Copy only the key string as shown in the sample below.
 
         ![sample-key-string]({{base_path}}/assets/img/guides/sample-key-string.png)
+   
+    5. Paste the copied private key in the Client Private Key text area.
+   
+    6. Click **Decrypt** and the details of the decrypted ID Token will be displayed.
 
-4. The following sample JAVA program can be used to decrypt the ID token using the default `wso2carbon.jks` keystore.
```
Comment (somindatommy) [inline en/docs/guides/login/oidc-token-encryption.md:74]: Why was this removed?

Comment (AnjanaSamindraPerera) [reply en/docs/guides/login/oidc-token-encryption.md:74]: This was mentioned [here](https://is.docs.wso2.com/en/6.0.0/guides/login/oidc-token-decryption/) and this section only talks about [decrypting it using the WSO2 IS playground sample application](https://is.docs.wso2.com/en/6.0.0/guides/login/oidc-token-encryption/#try-it).
